student.rb

```diff
@@ -0,0 +1,11 @@
+require './person'
+class Student < Person
+  def initialize(classroom)
+    super
```

Comment:
- [ ] hello an excellent job but the super method is one of the keys here hint: ( The code will result in a Student object that has the following attributes: age: The age of the student. classroom: The classroom the student attends. name: The name of the student. parent_permission: Whether or not the parent is allowed to see their child's grades.)